docs/embassy-deployment.md

```diff
@@ -16,9 +16,9 @@
     * Create the necessary number of floating IPs (one for each instance you would like to make publicly accessibly). Use the external network which you identified on the previous step.
 1. **Security group.**
     * Go to Compute → Access & Security → Security Groups.
-    * Click on “Create Security Group”. Name = “SSH-and-ping”, description = “Allows inbound SSH and ICMP access”.
+    * Click on “Create Security Group”. Name = “Web app”, description = “Allows inbound SSH and ICMP access. Opens ports 80, 443, and 8000.”.
     * Click on “Manage rules” for that security group.
-    * Add the rules for “SSH” and “ALL ICMP”.
+    * Add the rules for: “SSH”, “ALL ICMP”, “HTTP”, “HTTPS”, and “Custom TCP Rule” with Port=8000.
```

Comment:
It's a bit strange that these changes show up here, given that they have already been merged into master. Since we have three PRs, stacked on top of each other, it might help to rebase all of them against master and each other to make sure we don't have any branch conflicts

Reply:
I think that is because the base branch for this PR is 'datasources_page' not 'master', so I am pretty sure that if we change that it should be fine. That being said, these were all great suggestions for the already merged deployment instructions so should I keep them and commit them in a separate small PR?

Reply:
> That being said, these were all great suggestions for the already merged deployment instructions so should I keep them and commit them in a separate small PR?

Wow, because of how GitHub keeps displaying changes in its strange manner, I didn't even realise those changes were already merged! Sorry about the confusion.

There's no need to do a separate PR, I'll just introduce those changes directly to the `master` branch myself, as they're trivial.